lte/gateway/docker/mme/Dockerfile.ubuntu18.04

```diff
@@ -382,4 +384,5 @@ RUN sed -i -e "s@echo_error@echo@" -e "s@echo_success@echo@" -e "s@echo_warning@
 WORKDIR /magma-mme
 RUN openssl rand -out /root/.rnd 128
 
-# CMD []
+# For the moment, let have a dummy command
+CMD ["sleep", "infinity"]
```

Comment:
⚠️  Watch out. `sleep infinity` doesn't handles SIGINT properly. So when you'll `docker stop` you can end up with delay/not stopping. I bumped into that issue a few month ago and the "recommended hack" was https://github.com/docker/compose/issues/1926#issuecomment-505294443

Reply:
@tmdzk 

That is just temporary. I will add entrypoint script and start automatically `sctpd` and `oai_mme` executables.

But thanks for the info. I don't use compose and never do `docker stop` --> directly `docker rm -f`

Raphael